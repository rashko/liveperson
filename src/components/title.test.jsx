import React from "react";
import { shallow } from "enzyme";
import Title from "./title";

describe("Title", () => {
  it("should render correctly with no props", () => {
    const component = shallow(<Title />);
    expect(component).toMatchSnapshot();
  });

  it("should render correctly with props", () => {
    const title = "this is a test";
    const component = shallow(<Title title={title} />);
    expect(component).toMatchSnapshot();
  });
});
