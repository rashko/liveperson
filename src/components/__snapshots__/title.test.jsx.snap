// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly with no props 1`] = `
<styled.div>
  <styled.div />
  <styled.div />
</styled.div>
`;

exports[`Title should render correctly with props 1`] = `
<styled.div>
  <styled.div />
  <styled.div>
    this is a test
  </styled.div>
</styled.div>
`;

exports[`should render correctly with no props 1`] = `
<styled.div>
  <styled.div />
  <styled.div />
</styled.div>
`;

exports[`should render correctly with props 1`] = `
<styled.div>
  <styled.div />
  <styled.div>
    this is a test
  </styled.div>
</styled.div>
`;
